### PR TITLE
#40 Add About this Project section in readme

### LINKED ISSUE #40: Update about this project section
Update `README.md` with a 'about this project' section
- Change title to Soengram
- Split up `Object / Description`
- Add `Features`

### PR DESCRIPTION
**Pull request by domsec**
- Update title to reflect project name change
- Moved `Objective / Description` and added `Features` under `About this Project`
- Updated `DB_DATABASE` in `.env.sample` to reflect project name change

close #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+# Soengram
 
-## Objectives / Description
+> A photo sharing social networking website
 
-The goal of this app is to allow our users to post photos and interact with others. Our users are anyone who wants to use social media to interact with others. With features such as following and commenting we allow for a lot of interaction. A user can signup/login and immediately start posting, following, and commenting.
+## About this Project
 
-## Features
+Social media platforms provide instant gratification and analytical data among users - businesses, celebrities, influencers and everyday individuals - as they are able to gauge engagement and impact through their posts and comments received by other users. Follower count is a measure by which users are gauge their popularity among other users on the platform. Soengram provides users all these tools by allowing users to easily interact with others through posting photos on their account timeline and spark conversation through commenting on posts. Users are also able to stay up to date and informed by following friends, family and favorite celebrities and influencers.
```
Comment (mnashat) [inline README.md]:
```suggestion
Social media platforms provide instant gratification and analytical data among users - businesses, celebrities, influencers and everyday individuals - as they are able to gauge engagement and impact through their posts and comments received by other users. Follower count is a measure by which users can gauge their popularity among other users on the platform. Soengram provides users all these tools by allowing users to easily interact with others through posting photos on their account timeline and spark conversation through commenting on posts. Users are also able to stay up to date and informed by following friends, family and favorite celebrities and influencers.
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 
-Our instagram clone is a modern web app which gives users full functionality to post, follow, comment and more. There is an intuitive user interface from the signup, to the profile, to the feed. A full feature list is as follows:
+### Objective
+
+Develop a photo sharing social networking website where businesses, celebrities, influencers and regular everyday individuals can post photos, commnet on posts and follow other users.
```
Comment (mnashat) [inline README.md]:
```suggestion
Develop a photo sharing social networking website where businesses, celebrities, influencers and regular everyday individuals can post photos, comment on posts and follow other users.
```

**Review by lealakkis — DISMISSED** [review]

**Review by mnashat — CHANGES_REQUESTED** [review]
Kindly modify the marked words/sentences.

File: README.md (hunk 2 above)
Comment (domsec) [reply README.md]: Typos fixed in new commit

File: README.md (hunk 1 above)
Comment (domsec) [reply README.md]: Typos fixed in new commit

**Review by mnashat — APPROVED** [review]

**Review by jason-gerard — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mnashat) [reply README.md]: Resolved

**Pull request merged**